### DISCUSSION TITLE
Help 5 plots in phase 5 but destination drive full

### DISCUSSION THREAD

**Discussion by Kali123411** [swar/Swar-Chia-Plot-Manager#387] [General]
I had 5 plots that made it to phase 5 last night but the drive got full. I tried changing the destination drive in the job but it appears it still is trying to copy to the old location. Is there any way to fix this?

**Comment by raid5** [comment]
You'll probably have to copy the 101gb packed tmp files to your destination manually and change the extension to `.plot`

**Reply by Kali123411** [reply]
have you done that before?

**Reply by Matze5800** [reply]
I did, works fine.

**Reply by raid5** [reply]
Yup I've manually copied them over before
